views/update_line_notify_access_token.erb

```diff
@@ -0,0 +1,20 @@
+<ol class="breadcrumb">
+  <li><a href="/">Home</a></li>
+  <li><a href="<%= to("/project/#{@project.id}") %>"><%= @project.name %></a></li>
+  <li class="active">Update project</li>
+</ol>
+
+<div class="page-header">
+  <h1>Update project configuration - <%= @project.name %></h1>
+</div>
+
+<div class="row">
+  <form method="post" action="<%= to("/project/#{@project.id}/update_line_notify_access_token") %>">
+    <div class="form-group">
+      <label for="inputLineNotifyAccessToken">LINE Notify Access Token</label>
+      <input type="password" name="line_notify_access_token" class="form-control"  id="inputLineNotifyAccessToken" value="<% if @project.line_notify_access_token %>***<% end %>">
```

Comment:
I think things look good, separating things out I think is the right idea, since it makes things a bit easier to understand. You should remove the value="" part here, and probably assume if a user makes it to this page, they either want to clear the token (submit an empty value) or submit a new one. Adding a default value here means that if a user navigates here and submits, then they would set their token to "***" which is probably not what they intended to do.

```
<input type="password" name="line_notify_access_token" class="form-control"  id="inputLineNotifyAccessToken">

```

should be sufficient